src/convertToObject.js

```diff
@@ -7,6 +7,29 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. Consider using a name that reflects the content of the variable, such as `stylesObject` or `cssProperties`.